feathr_project/feathr/utils/job_utils.py

```diff
@@ -38,16 +40,30 @@ def get_result_df(client: FeathrClient, format: str = None, res_url: str = None,
             delta = DeltaTable(local_dir_path)
             if not client.spark_runtime == 'azure_synapse':
                 # don't detect for synapse result with Delta as there's a problem with underlying system
-                # Issues are trached here: https://github.com/delta-io/delta-rs/issues/582
+                # Issues are tracked here: https://github.com/delta-io/delta-rs/issues/582
                 result_df = delta.to_pyarrow_table().to_pandas()
             else:
-                logger.info("Please use Azure Synapse to read the result in the Azure Synapse cluster. Reading local results is not supported for Azure Synapse. Emtpy DataFrame is returned.")
+                logger.info("Please use Azure Synapse to read the result in the Azure Synapse cluster. Reading local results is not supported for Azure Synapse. Empty DataFrame is returned.")
                 result_df = pd.DataFrame()
         elif format.casefold()=="avro":
             import pandavro as pdx
             for file in glob.glob(os.path.join(local_dir_path, '*.avro')):
                 dataframe_list.append(pdx.read_avro(file))
             result_df = pd.concat(dataframe_list, axis=0)
+        elif format.casefold()=="csv":
+            for file in glob.glob(os.path.join(local_dir_path, '*.csv')):
+                try:
+                    df = pd.read_csv(file, index_col=None, header=None)
+                except EmptyDataError:
+                    # in case there are empty files
+                    df = pd.DataFrame()
+                dataframe_list.append(df)
+            result_df = pd.concat(dataframe_list, axis=0)
+            # Reset index to avoid duplicated indices
+            result_df.reset_index(drop=True)
+        else:
+            raise RuntimeError(f"{format} is currently not supported in get_result_df. Please consider writing a customized function to read the result.")
+
     else:
```

Comment:
Will this `else` clause still useful given the above `RuntimeError`?

Reply:
yes, since if `format` is not set, by default Feathr will read avro files